app-benchmarks/stress-ng/files/makefile.patch

```diff
@@ -0,0 +1,75 @@
+--- a/Makefile	2018-04-27 17:02:16.000000000 +0200
++++ b/Makefile	2018-05-05 00:15:00.687399505 +0200
+@@ -21,7 +21,7 @@
+ # Codename "portable pressure producer"
+ #
+ 
+-CFLAGS += -Wall -Wextra -DVERSION='"$(VERSION)"' -O2 -std=gnu99
++CFLAGS += -DVERSION='"$(VERSION)"'
+ 
+ #
+ # Pedantic flags
+@@ -314,12 +314,10 @@
+ .o: stress-ng.h Makefile
+ 
+ .c.o: stress-ng.h Makefile $(SRC)
+-	@echo "CC $<"
+-	@$(CC) $(CFLAGS) -c -o $@ $<
++	$(CC) $(CFLAGS) $(LDFLAGS) -c -o $@ $<
+ 
+ stress-ng: info $(OBJS)
+-	@echo "LD $@"
+-	@$(CC) $(CPPFLAGS) $(CFLAGS) $(OBJS) -lm $(LDFLAGS) -lc -o $@
++	$(CC) $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) $(OBJS) -lm $(LDFLAGS) -lc -o $@
```

Comment:
You have LDFLAGS twice. `-lc` seems unnecessary.

Reply:
Whoops, seems I didn't saw that. Thanks.